build/Settings.cs

```diff
@@ -67,7 +69,11 @@ private static string config(string @default = null, [CallerMemberName] string k
                     { "win-x86", _winPowershellRuntimes },
                     { "win-x64", _winPowershellRuntimes },
                     { "linux-x64", new [] { "linux", "linux-x64", "unix", "linux-musl-x64" } },
-                    { "osx-x64", new [] { "osx", "osx-x64", "unix" } }
+                    { "osx-x64", new [] { "osx", "osx-x64", "unix" } },
+                    // NOTE: arm support wasn't added until PowerShell 7.2
```

Comment:
Could you please update the comment to mention that PowerShell 7.0 does not have ARM support?